newsroom/products/products.py

```diff
@@ -0,0 +1,67 @@
+import newsroom
+import superdesk
+
+
+class ProductsResource(newsroom.Resource):
+    """
+    Products schema
+    """
+    schema = {
+        'name': {
+            'type': 'string',
+            'unique': True,
+            'required': True
+        },
+        'description': {
+            'type': 'string'
+        },
+        'sd_product_id': {
+            'type': 'string'
+        },
+        'query': {
+            'type': 'string'
+        },
+        'is_enabled': {
+            'type': 'boolean',
+            'default': True
+        },
+        'product_type': {
+            'type': 'string',
+            'allowed': ['query', 'superdesk', 'curated']
```

Comment:
why types? wouldn't it be easier to config if you could combine superdesk id with a query?